### DISCUSSION TITLE
The aside off-canvas view can automatically hide

### DISCUSSION THREAD

**Discussion by 779575267** [anapnoe/stable-diffusion-webui-ux#125] [Ideas]
The aside off-canvas view and can automatically hide or be hidden by clicking outside the frame,Just like the menu in the left-side title bar.
really like your interface. Thank you so much.

**Comment by anapnoe** [comment]
it was made like this so you can go to prompt and edit without breaking your workflow as when you start to type something in the prompt it will close the extra networks maybe now that I think of it again you can click all except parent and prompts 
you can post this in issues as an enhancement  you are welcome
